inginious/frontend/plugins/multilang/static/multilang.js

```diff
@@ -168,16 +168,16 @@ function toggle_display_new_subproblem_option() {
     else new_subproblem_element.show();
 }
 
-const render_notebook = function (ipynb) {
-    const notebook_holder = $("#notebook-holder")[0];
-    $("#notebook-holder").hide();
+const render_notebook = function (ipynb, notebook_holder_id) {
```

Comment:
Probably call the variable `notebook_holder_element`, as this receives the element itself, rather than just the element id.